storybook/stories/Navbar.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react"
+import { BrowserRouter } from "react-router-dom"
+import { action } from "@storybook/addon-actions"
+import { Navbar, Environment } from "@misk/core"
+
+export const Production = () => (
+  <BrowserRouter>
+    <Navbar
```

Comment:
Really powerful to see how simple it is to build a new story to test out a UI component. Pretty excited at the thought of adding the rest of Misk-Web components here and potentially transitioning it to be the primary docs site.